oak/client/manager_client.h

```diff
@@ -54,13 +58,14 @@ class ManagerClient {
     request.set_allocated_application_configuration(application_config.release());
 
     LOG(INFO) << "Creating Oak Application";
-    grpc::Status status = stub_->CreateApplication(&context, request, &response);
+    grpc::Status status = stub_->CreateApplication(&context, request, response.get());
     if (!status.ok()) {
-      LOG(QFATAL) << "Failed: " << status.error_code() << '/' << status.error_message() << '/'
-                  << status.error_details();
+      LOG(ERROR) << "Failed: " << status.error_code() << '/' << status.error_message() << '/'
```

Comment:
This is moving the ownership for error behavior one level above. Is there any case where everything works in case we fail to create an application?

Reply:
Not with the current examples &ndash; no app, no example.